base-images-sbom-script/app/requirements.txt

```diff
@@ -0,0 +1 @@
+packageurl-python==0.14.0
```

Comment:
Could we use the `requirements.in` + `requirements.txt` approach (generate txt from in via `pip-compile`) to ensure all our dependencies are pinned?

Reply:
updated